index.html

```diff
@@ -66,26 +73,29 @@ <h2>I'm Jay</h2>
     </div>
   </section>
   <section class="works_container">
-    <div class="works_title">
+    <div id="works_title">
       <h2 class="name_title">My Recent Works</h2>
```

Comment:
- [ ] Kindly add the appropriate color to **_`My recent Works`_** element as specified in the Figman design you chose and remove the line breaks so that it matches the Figma design chosen 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/186531626-fe3630b3-708e-4481-9db2-6e7d70b0c3e9.png)  | ![image](https://user-images.githubusercontent.com/63473584/186531675-5d0ce78b-bf9b-4065-93a5-c2ef6f126d34.png)  |